add-matcha.php

```diff
@@ -0,0 +1,56 @@
+<?php
+//connect to the database and hide the db-connection.php into gitignore
+//get products from the database by creating a separate file for functions
+require_once 'db-connection.php';
+require_once 'functions.php';
+$db = getDb();
+
+if(isset($_POST['submit']))
+{
+    $image = $_POST['image'];
+    $productName = $_POST['product_name'];
+    $businessName = $_POST['business_name'];
+    $currency = $_POST['currency'];
+    $price = $_POST['price'];
+    $sql = "INSERT INTO products (image, product_name, business_name, currency, price)
+     VALUES ('$image', '$productName', '$businessName','$currency', '$price')";
+    if (mysqli_query($conn, $sql)) {
+        echo "New record has been added successfully !";
+    } else {
+        echo "Error: " . $sql . ":-" . mysqli_error($conn);
+    }
+    mysqli_close($conn);
+}
+
+?>
+
+<!DOCTYPE html>
```

Comment:
Don't think you use this in .php files?

Reply:
Deleted DOC type... I did not know that but now I do!

Reply:
I have also deleted it in my index.php file.